directus-okta-sso/index.md

```diff
@@ -0,0 +1,177 @@
+| Audience URI (SP Entity ID) | http://localhost:8055/auth/login/okta/metadata              | A route in our Directus application that provides metadata to verify that our application is the intended recipient of the SAML response   |
+| Default RelayState          | http://localhost:8055/admin/login?reason=SIGN_OUT&continue= | A route in our Directus application to which the user will be redirected after successful authentication                                   |
+
+Note: Use `localhost:8055` for local development. For deployment to other domains, change values to your domain, which will most likely work over the https protocol (e.g., `https://example.com/...`). Alternatively, we can create as many Okta applications as needed for each environment.
+
+Next, fill in the **Attribute Statements**.
+Note: Although it is labelled as optional, it is critical to fill in at least the email attribute for authentication to work.
+
+The attributes will look like this:
+
+| Name       | Name format | Value          | Description                                                                                                                                                                      |
+|------------|-------------|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| email      | Basic       | user.email     | An important field by which Directus will match the email from our Okta account with the email in our app                                                                        |
+| first_name | Basic       | user.firstName | The field by which Directus will match the first name from the Okta account with the first name in our app. Add it if we want the user's first name to be pre-filled in the app. |
+| last_name  | Basic       | user.lastName  | The field by which Directus will match the last name from the Okta account with the last name in our app. Add it if we want the user's last name to be pre-filled in the app.    |
+
+
+Let's leave the rest of the parameters at this step as defaults. As a result, we will get parameters like these:
+
+![Configure general SAML settings](configure-saml-screen-settings.png)
+
+![Configure SAML attributes](configure-saml-screen-attributes.png)
+
+Click **Next** and go to the third and final screen for the Okta app settings.
+
+### Feedback
+Our setup is complete, and we are prompted to leave feedback or submit the app to Okta for review. Unless we want to spend any more time here, select **I'm an Okta customer adding an internal app** and, without filling anything in, click **Finish**.
+
+![Process feedback screen](feedback-screen.png)
+
+### Add Users
+Now, we can choose which people from our organization can access Directus through Okta SSO. Go to the **Assignments** tab in the settings of our Okta application.
```

Comment:
```suggestion
Choose which people from our organization can access Directus through Okta SSO. Go to the **Assignments** tab in the settings of our Okta application.
```

I've noticed `Now, ` be present a few times. This is more spoken than it is written. Always try and lead with instruction when technical writing for more concise pieces.